src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
```

Comment:
>FilingTargetType [](start = 22, length = 16)

Ryley points out (and now that i think of it, i agree) we could potentially derive this information from the root url, so do we really need it?

Reply:
You are right -- at least for now. You can imagine a system where customers can stand up their own instances ([https://www.example.com/issues](https://www.example.com/issues), [https://www.contoso.com/issues](https://www.contoso.com/issues)). Then you can't tell. But as long as we only support ADO and GitHub, we don't need this.

---
In reply to: [306568347](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306568347) [](ancestors = 306568347)

Reply:
Fixed.

---
In reply to: [306570111](https://github.com/microsoft/sarif-sdk/pull/1583#discussion_r306570111) [](ancestors = 306570111,306568347)